hooks/pre_gen_project.py

```diff
@@ -0,0 +1,33 @@
+import re
+import sys
+
+
+MODULE_REGEX = r"^[_a-zA-Z][_a-zA-Z0-9]+$"
+
+
+class bcolors:
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
```

Comment:
Nit: A number of these values are unused. They can be removed, but not a big deal.

Reply:
Thanks for the review!